_episodes/07-development-setup.md

```diff
@@ -201,31 +210,45 @@ It also gives other people an early opportunity to provide input and suggestions
 which results in more valuable contributions.
 
 Then, we create a new ``branch`` locally and start developing new codes.
+To create a new branch:
+~~~bash
+git checkout -b your_branch_name
+~~~
+
```

Comment:
nice to add a link to the https://tutorial.esmvaltool.org/setup.html where it is mentioned git is one of the prerequisites.